core/runtime/src/main/java/io/quarkus/runtime/configuration/ConfigUtils.java

```diff
@@ -98,6 +98,7 @@ public static SmallRyeConfigBuilder configBuilder(final boolean runTime, final b
         if (runTime) {
             builder.addDefaultSources();
             builder.withSources(dotEnvSources(classLoader));
+            builder.withSources(new UUIDConfigSource());
```

Comment:
An alternative way we could use is to call `builder.withDefault`.  This will add a property with a value to the defaults source which was a priority of `Integer.MIN_VALUE`.

Reply:
Didn't know that!

I've switched to this approach and updated the configuration.
So no more additional config source.